go.mod

```diff
@@ -214,6 +214,8 @@ require (
 	sigs.k8s.io/structured-merge-diff/v4 v4.4.1 // indirect
 )
 
+require github.com/openshift/node-feature-discovery/api/nfd v0.0.0-20250103121957-354b94ed3e55
```

Comment:
Not sure why this is left outside the 2 `require` blocks above . Can you move it to the first one ?